about.html

```diff
@@ -0,0 +1,109 @@
+        <a class="thecd" href="https://www.amazon.com/You-Red-Hot-Chili-Peppers/dp/B0054N73EY/ref=sr_1_6?crid=21AL2RXMI90ZM&keywords=im+with+you+cd&qid=1649808338&sprefix=im+with+yu+cd%2Caps%2C165&sr=8-6"><img src="./assets/Images/album9.jpg" alt="im with you"></a>
+        <a class="thecd" href="https://www.amazon.com/Stadium-Arcadium-2CD-Chili-Peppers/dp/B000EMGAOY/ref=sr_1_1?crid=27B07IY478AR6&keywords=stadium+arcadium+cd&qid=1649808305&sprefix=stadium+arcadiumcd%2Caps%2C221&sr=8-1"><img src="./assets/Images/album8.jpg" alt="stadium arcadium"></a>
+        <a class="thecd" href="https://www.amazon.com/Way-Red-Hot-Chili-Peppers/dp/B000067CPX/ref=sr_1_2?crid=NJUK25R28D4O&keywords=by+the+way+cd&qid=1649808283&sprefix=by+the+wa+cd%2Caps%2C162&sr=8-2"><img src="./assets/Images/album7.jpg" alt="by he way"></a>
+        <a class="thecd" href="https://www.amazon.com/Californication-Red-Chili-Peppers-Audio/dp/B00FT15NXE/ref=sr_1_1?crid=R5EQR2D2RTGB&keywords=californication+cd&qid=1649808262&sprefix=californication+cd%2Caps%2C159&sr=8-1"><img src="./assets/Images/album6.jpg" alt="californication"></a>
+        <a class="thecd" href="https://www.amazon.com/One-Hot-Minute-Chili-Peppers/dp/B000002MTR/ref=sr_1_1?crid=2DXXICCNCFZ61&keywords=one+hot+minute+cd&qid=1649808240&sprefix=one+hot+minute+c%2Caps%2C180&sr=8-1"><img src="./assets/Images/album5.jpg" alt="one hot minute"></a>
+        <a class="thecd" href="https://www.amazon.com/Blood-Sugar-Magik-Chili-Peppers/dp/B000002LQR/ref=sr_1_1?keywords=blood+sugar+sex+magik+cd&qid=1649808199&sprefix=blood+sugar+se%2Caps%2C210&sr=8-1"><img src="./assets/Images/album4.jpg" alt="blood sugar sex magik"></a>
+        <a class="thecd" href="https://www.amazon.com/Mothers-Milk-Explicit-Chili-Peppers/dp/B000078DOI/ref=sr_1_1?crid=S2G0VZ0O8VEZ&keywords=mothers+milk+cd&qid=1649858489&sprefix=mothers+milk+cd%2Caps%2C186&sr=8-1"><img src="./assets/Images/albumX.jpg" alt="mothers milk"></a>
+        <a class="thecd" href="https://www.amazon.com/Uplift-Mofo-Party-Plan-Explicit/dp/B000078DOJ/ref=sr_1_1?keywords=uplift+mofo+party+plan+cd&qid=1649808151&sprefix=uplift+mofo%2Caps%2C178&sr=8-1"><img src="./assets/Images/album3.jpg" alt="the uplift mofo party plan"></a>
+        <a class="thecd" href="https://www.amazon.com/Freaky-Styley-Explicit-Chili-Peppers/dp/B000078DOH/ref=sr_1_1?keywords=freaky+styley+cd&qid=1649808126&sprefix=freaky+sty%2Caps%2C170&sr=8-1"><img src="./assets/Images/album2.jpg" alt="freaky styley"></a>
+        <a class="thecd" href="https://www.amazon.com/Red-Hot-Chili-Peppers-Explicit/dp/B000078DOK/ref=sr_1_10?crid=20AICI1OMUODH&keywords=the+red+hot+chili+peppers+album&qid=1649808086&sprefix=the+red+hot+chili+peppers+albu%2Caps%2C189&sr=8-10"><img src="./assets/Images/album1.jpg" alt="self-titled"></a>
```

Comment:
- [ ] Kindly you can change the content of the original design, but you have to stick to the layout.
- [ ] Kindly you can change these photos with two pictures and put text centered over them.


Your design
![image](https://user-images.githubusercontent.com/35535130/163291494-cf094734-7b9a-4273-8c07-84ccb60fe230.png)

Original design
![image](https://user-images.githubusercontent.com/35535130/163292070-81dff331-0381-41ed-ad2f-878b75c7d14f.png)

Reply:
Just finished this part and i think I'm done with the changes! thank you so much for your kind feedback!